### PR TITLE
PIM-2938: Handle array values in NormalizeConfigurationExtension

### PR DESCRIPTION
**Pull request by srascar**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | - |
| Fixed tickets | #2938 |
| Checkstyle issues?* | yes |
| Doc PR | - |

### CODE REVIEW

**Comment by sumbobyboys** [comment]
Thanks for your PR, I will review it

File: src/Pim/Bundle/ImportExportBundle/Twig/NormalizeConfigurationExtension.php [hunk 1]
```diff
@@ -75,10 +77,33 @@ public function normalizeValueFilter($value)
...
 
     /**
+     * Normalize an array value to print intelligible data to user
+     *
+     * @param array $value
+     * @param array $labels
+     *
+     * @return string
+     */
+    protected function normalizeArrayValue(array $value, array $labels = array())
```
Comment (sumbobyboys) [inline src/Pim/Bundle/ImportExportBundle/Twig/NormalizeConfigurationExtension.php]: we use the [] notation for arrays

Comment (srascar) [inline src/Pim/Bundle/ImportExportBundle/Twig/NormalizeConfigurationExtension.php]: @sumbobyboys  done.

**Comment by grena** [comment]
Hello @srascar , thanks again for this contribution :)

Could you please update your PR to match changes on master? This way, we'll run integration tests in order to merge it!

**Comment by srascar** [comment]
@grena PR updated

**Comment by grena** [comment]
@srascar Thank you! :ok_hand:

**Comment by jjanvier** [comment]
GTM

**Comment by pierallard** [comment]
Hi @srascar 

Thanks a lot for your contribution :tada: !

I get the results of the CI, and there is 2 failing behats:
- features/export/edit_an_export.feature:20
- features/import/edit_an_import.feature:20
The error is raised because you changed the previous behavior. Before, when you display the import/export job, we saw "decimal separator: comma (,)". Now, with your updates, it displays "decimal separator: ,". Can you try to fix it before we merged your contribution ? Don't hesitate any second if you want help to fix it, we're here to carry contributions ;)

Pierre

**Comment by SamirBoulil** [comment]
PR is here https://github.com/akeneo/pim-community-dev/pull/2946

**Comment by pierallard** [comment]
Hi @srascar 
We will close this PR, because the code you edited 1 year ago was now deprecated on master version.
Feel free to open an issue on our board if the issue is still present on latest version!
Regards

Pierre

**Pull request closed without merging**